tests/bundle/test_bundle.py

```diff
@@ -1,31 +1,42 @@
 import pytest
 
 from tests.types import UserOperation, RPCErrorCode
-from tests.utils import assertRpcError, dumpMempool, deploy_wallet_contract
+from tests.utils import (
+    assertRpcError,
+    dumpMempool,
+    deploy_wallet_contract,
+    deploy_and_deposit,
+)
 
 
 @pytest.mark.parametrize("mode", ["manual"], ids=[""])
 @pytest.mark.usefixtures("clearState", "setBundlingMode")
 def test_bundle_replace_by_fee(w3):
     wallet = deploy_wallet_contract(w3)
     callData = wallet.encodeABI(fn_name="setState", args=[1])
+    lowerFeePerGas = hex(10**9)
+    higherFeePerGas = hex(10**10)
+    midFeePerGas = hex(5 * 10**9)
     lowerFeeOp = UserOperation(
         sender=wallet.address,
         nonce="0x1",
         callData=callData,
-        maxPriorityFeePerGas=hex(10**9),
+        maxPriorityFeePerGas=lowerFeePerGas,
```

Comment:
It's probably best to separate it to two different tests, one for maxFeePerGas and another for maxPriorityFeePerGas